site/_posts/2017-01-30-keeping-up-with-the-conferences.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Last Week in ManageIQ: Keeping up with the conferences"
+author: Dávid Halász
+date: 2017-01-30
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi, I am David from the ManageIQ UI team. I will be your guide in this quick recap of what happened in the last week on our premises.
+
+## Featured
+During the weekend, we were promoting ManageIQ on [DevConf.cz](http://devfonf.cz), an amazing conference held in Brno, Czech Republic. Mooli Tayer from the container management team gave a really interesting talk about [automating a datacenter using ManageIQ and Ansible](https://www.youtube.com/watch?v=oK8l_u_kHC8). To learn more, check out our [twitter feed](https://twitter.com/manageiq) and look for the [#DevConfCZ](https://twitter.com/search?l=&q=%23devconfcz%20from%3Amanageiq&src=typd) hashtag.
```

Comment:
Typo: http://devconf.cz/

For the hashtag link, consider removing "from:" so it includes tweets tweeted by others (e.g. you :smile: )